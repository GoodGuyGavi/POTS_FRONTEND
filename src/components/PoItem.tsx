import React from "react";
import { ListGroup, ListGroupItem } from "react-bootstrap";

export interface IPoItemProps {
  poStatus?: any;
  poNumber?: any;
}

const PoItem = (props: IPoItemProps) => {
  const { poStatus, poNumber } = props;
  return (
    <div>
      <h6>{poNumber}</h6>
      <h6>Status: {poStatus}</h6>
    </div>
  );
};

export default PoItem;
