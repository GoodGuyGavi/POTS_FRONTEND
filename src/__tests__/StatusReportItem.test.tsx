/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
import PoItem from "../components/PoItem";
import StatusReportList from "../components/Status";

const props = {
  status: [
    {
      id: 1,
      number: "3",
      statOption: "Orders Received"
    },
    {
      id: 2,
      number: "7",
      statOption: "Released from supplier"
    },
    {
      id: 3,
      number: "2",
      statOption: "On-hold from supplier"
    }
  ]
};

describe("Status Report Item", () => {
  test("renders correctly", () => {
    expect(shallow(<StatusReportList {...props} />)).toMatchSnapshot();
  });
});
