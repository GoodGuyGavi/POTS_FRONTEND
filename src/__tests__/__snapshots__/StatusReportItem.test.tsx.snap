// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status Report Item renders correctly 1`] = `ShallowWrapper {}`;
